### PR TITLE
Fixes RHICOMPL-102: Add widget-id to policy cards

### PR DESCRIPTION
**Pull request by akofink**
This aids front end test automation.

### CODE REVIEW

File: src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ class CompliancePolicyCard extends React.Component {
         );
 
         return (
-            <Card>
+            <Card widget-id={this.policy.id}>
```
Comment (quarckster) [inline src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: Thanks! That exactly what I asked!

Comment (quarckster) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: What kind of id is stored in `this.policy.id`?

Comment (akofink) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: It looks like this: `5ace0d15-178b-4624-93ea-306aec38f24e`. I think it's the UUID stored in our database.

Comment (quarckster) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: Hm, is it possible to put a value that would be related to name of the card?

Comment (akofink) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: The id needs to be unique, right? The policy name isn't unique here. Here's an example of what we have access to:

image

Comment (dLobatog) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: `ref_id` is unique in the db too, for each account

Comment (akofink) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: Here's an example of duplicate policies in the UI. I'm not sure if this is a bug or if it's intentional.

image

Comment (quarckster) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: For me looks like a bug

Comment (akofink) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: One of these systems is RHEL7; the other is CentOS7. Perhaps that's causing it?

Comment (akofink) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: @quarckster would you like me to use `ref_id`? I think that's a good idea.

Comment (quarckster) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: Probably, and I think it's quite important questions, should we handle centos reports at all?

Comment (akofink) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: The ref_id is different for the two policy cards shown in the screenshot above. rhel7 shows ref_id of xccdf_org.ssgproject.content_profile_standard. centos7 shows ref_id of standard.

Comment (akofink) [reply src/SmartComponents/CompliancePolicyCard/CompliancePolicyCard.js]: No, we don't officially support centos or fedora, but they will still work, I believe.

**Comment by akofink** [comment]
Updated to use ref_id!

**Pull request merged**